api/run_tests.sh

```diff
@@ -13,6 +13,12 @@ cd $script_directory
 # move up a level
 cd ..
 
+# Ensure that postgres is running
+if ! docker ps | tail -n +2 | awk '{ print $NF }' | grep drdb > /dev/null; then
```

Comment:
I suppose you are using this command to test whether `drdb` container is running or not. This is probably better:
`docker ps --filter NAME=drdb -q`
If the command above gives nothing, `drdb` is not running.